package.json

```diff
@@ -1,22 +1,44 @@
 {
-  "name": "pizza1",
-  "version": "1.0.0",
-  "description":"CYBERPIZZA",
-  "main": "index.js",
-  "scripts": {
-    "start": "node server"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/AStashevskaya/cyberpizza.git"
-  },
-  "author": "",
-  "license": "ISC",
-  "bugs": {
-    "url": "https://github.com/AStashevskaya/cyberpizza/issues"
-  },
-  "homepage": "https://github.com/AStashevskaya/cyberpizza#readme",
-  "dependencies": {
-    "express": "^4.17.1"
-  }
+    "name": "pizza1",
+    "version": "1.0.0",
+    "description": "CYBERPIZZA",
+    "main": "index.js",
+    "scripts": {
+        "start": "webpack serve",
+        "lient:watch": "webpack --mode development --watch --open",
```

Comment:
@AStashevskaya I think it should be `client:watch`, and could you doublecheck your `webpack` version and settings. After running `npm i` and `npm run lient:watch` it showed me error:

<img width="542" alt="image" src="https://user-images.githubusercontent.com/12561302/113323756-a0b96400-9316-11eb-8ccc-bc419944026a.png">